opencontrail/files/cassandra_backup

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -uef -o pipefail
+
+# Define vars
+BACKUP_DIR=/var/backups/cassandra
+DATA_DIR=/var/lib/cassandra/data
+NODETOOL=$(which nodetool)
+
+TODAY_DATE=$(date +%F)
+NODENAME=$(salt-call pillar.get linux:network:fqdn --out=newline_values_only)
+CONTROL_NODES_DB_TAG="OPENCONTRAIL_CONTROL_DB"
+BACKUP_SNAPSHOT_DIR="${BACKUP_DIR}/${CONTROL_NODES_DB_TAG}/${NODENAME}/${TODAY_DATE}/SNAPSHOTS"
+BACKUP_SCHEMA_DIR="${BACKUP_DIR}/${CONTROL_NODES_DB_TAG}/${NODENAME}/${TODAY_DATE}/SCHEMA"
+
+SNAPSHOT_NAME=snp-$(date +%F-%H%M-%S)
+DATE_SCHEMA=$(date +%F-%H%M-%S)
+
+DB_BIND_HOST=$(salt-call pillar.get opencontrail:database:bind:host --out=newline_values_only)
+DB_BIND_PORT=$(salt-call pillar.get opencontrail:database:bind:port --out=newline_values_only)
+CQL_SH="$(which cqlsh) ${DB_BIND_HOST} ${DB_BIND_PORT}"
+## List All Keyspaces
+declare -a KEYSPACE_LIST=( $( ${CQL_SH} -e "DESC KEYSPACES" | awk '{RS="\\s+"; if(NF>0){print}}' | sort ) )
+
+# Make sure backup Directory exists
+mkdir -p "${BACKUP_SCHEMA_DIR}"
```

Comment:
I would rather manage that dirs by backup.sls in opencontrail formula. (rights/content, etc..)